### PR TITLE
BO-82 "cm::services::stream" extend with status-secret token param

### PR DESCRIPTION
**Pull request by kris-lab**
Depends on https://github.com/cargomedia/bipbip/pull/175

### CODE REVIEW

**Comment by kris-lab** [comment]
@tomaszdurka please review

File: modules/cm/manifests/services/stream.pp [hunk 1]
```diff
@@ -1,10 +1,11 @@
-class cm::services::stream(
-  $port = 8090,
+class cm::services::stream (
+  $port         = 8090,
   $ssl_cert,
```
Comment (tomaszdurka) [inline modules/cm/manifests/services/stream.pp]: Note: should you order params by non-optonial in front?

File: modules/socket_redis/manifests/init.pp [hunk 2]
```diff
@@ -1,9 +1,10 @@
...
-  $version = 'latest',
-  $redisHost = 'localhost',
+  $version     = 'latest',
+  $redisHost   = 'localhost',
   $socketPorts = [8090],
-  $logDir = '/var/log/socket-redis',
-  $statusPort = '8085',
+  $logDir      = '/var/log/socket-redis',
+  $statusPort  = '8085',
+  $statusToken = 'supersecret',
```
Comment (tomaszdurka) [inline modules/socket_redis/manifests/init.pp]: This is optional param, let's not make it mandatory here?
image

File: modules/cm/manifests/services/stream.pp [hunk 3]
```diff
@@ -1,10 +1,11 @@
...
   $ssl_key,
-  $redis_host = '127.0.0.1',
+  $redis_host   = '127.0.0.1',
   $socket_ports = [8091, 8092, 8093, 8094],
-  $status_port = 8085
+  $status_port  = 8085,
+  $status_token = 'supersecret',
```
Comment (tomaszdurka) [inline modules/cm/manifests/services/stream.pp]: I would make it optional here too, but as you wish.

File: modules/cm/manifests/services/stream.pp (hunk 1 above)
Comment (kris-lab) [inline modules/cm/manifests/services/stream.pp]: I am OK to have it optional. I thought you wanna enforce this token for higher security.

Comment (tomaszdurka) [inline modules/cm/manifests/services/stream.pp]: Well, tbh. I am just describing possibilities from socket-redis library itself.
I guess we could have done that there. 

On the other hand I think it's nice to be able to start this library locally without any extra configuration.

**Comment by kris-lab** [comment]
@tomaszdurka please re-review, all is optional.

**Comment by kris-lab** [comment]
@tomaszdurka please re-review again, there was missing secret config for daemon!

**Comment by tomaszdurka** [comment]
ok, but still failing

**Comment by kris-lab** [comment]
retest this please

**Comment by kris-lab** [comment]
@tomaszdurka specs are fine, CI was having troubles again.

**Comment by tomaszdurka** [comment]
lgtm then

**Pull request merged**